community/kernel/src/test/java/org/neo4j/test/subprocess/SubProcess.java

```diff
@@ -214,7 +197,7 @@ private String classPath( String parentClasspath )
 
     private static Process start( String... args )
     {
-        ProcessBuilder builder = new ProcessBuilder( args );
+        ProcessBuilder builder = new ProcessBuilder( args ).inheritIO();
```

Comment:
Yes, have talked with @thobe , he does not remember exactly which tests do not allow us to made that change.
So far have tried to run IT, they looks fine.
Lets see what exactly fails...if its still the case.